### DISCUSSION TITLE
Titles aren't switching from the one onto the next while playing the preview

### DISCUSSION THREAD

**Discussion by iTucky** [SubtitleEdit/subtitleedit#7871] [Q&A] [answered]
Hello guys, I've started having a weird issue lately, idk how to tell about this properly, but hopefully you'll understand

So while I'm editing the subs, it was always playing the next line after the first one ends, but now it doesn't work

Is there a way to fix it? Can't find anything about it in the settings :/

P.S. attached a video of it for my issue being more understandable

https://github.com/SubtitleEdit/subtitleedit/assets/39695663/2ab44b7c-c9e2-4ae9-ba2d-e0421d255a76

**Comment by niksedk** [comment, marked as answer]
Perhaps this?
![image](https://github.com/SubtitleEdit/subtitleedit/assets/1261331/49655b73-c307-480d-8bd6-ab8b7fa410e8)

**Reply by iTucky** [reply]
oh really lol, thanks
